spec/person_spec.rb

```diff
@@ -0,0 +1,25 @@
+require './spec_helper'
+
+describe Person do
+  it 'should have a name, age, and rentals' do
+    person = Person.new(25, 'Alice')
+
+    expect(person.name).to eq('Alice')
+    expect(person.age).to eq(25)
+    expect(person.rentals).to be_an(Array)
+    expect(person.rentals).to be_empty
+  end
+
+  it 'should have an ID between 1 and 100' do
+    person = Person.new(25, 'Alice')
+
+    expect(person.id).to be_between(1, 100)
+  end
+
+  it 'should increment the count of people' do
+    initial_count = Person.count
+    Person.new(30, 'Bob')
+
+    expect(Person.count).to eq(initial_count + 1)
+  end
```

Comment:
- [ ] According to the project requirements, you're expected to write tests for all methods in the separate classes, and in this file, you left out the tests for the correct_name, can_use_services, add_rental, and the of_age methods. Kindly implement test for all methods in the separate files as required. 👍🏽